doc/community-build/docker/cortx-all/README.md

```diff
@@ -82,12 +83,12 @@ This document provides step-by-step instructions to build required binaries and
     docker run --name release-packages-server -v /var/artifacts/0/:/usr/share/nginx/html:ro -d -p 80:80 nginx
     ```
 
-7. Once docker container is up and running, We need to clone cortx repo use below commands for same.
+7. We need to clone cortx repo use below commands for same.
```

Comment:
s/repo use/repo, use/ 
(add a comma please btwn repo and use.

Reply:
@AbhijitPatil1992 Could you please address this comment